index.html

```diff
@@ -179,5 +179,43 @@ <h2 class="title-2">ROAD TRIP PLANNER</h2>
             </section>
         </section>
     </div>
+    <footer class="footer">
+        <section class="contact-1">
+                <h1 class="title-1">
+                    CONTACT US
+                </h1>
+                <p>
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. I’d love 
+                    to help with it 
+                </p>
+                <form class="contact-form" id="simple-form" name="contact_form" action="https://formspree.io/f/mvolqnaz" method="post">
+                    <fieldset id="form-inputs">
+                        <label for="full-name" hidden>Full Name</label>
+                        <input type="text" name="user_full_name" id="full-name" placeholder="Text Input" required="" maxlength="30">
+                        <label for="email-address" hidden="">Email Address</label>
+                        <input type="email" name="user_email" id="email-address" placeholder="jeremy@gmail.com" required="">
+                        <label for="message" hidden="">Message</label>
+                        <textarea rows="5" name="user_message" id="message" placeholder="Enter text here" required="" maxlength="500"></textarea>
+                    </fieldset>
+                    <div class="container-5">
+                        <button class="btn-1">See Project</button>
+                    </div>
+                </form>
```

Comment:
- [ ] Kindly include the 'See Project' button in the form. This button is supposed to be the submit button of this form.